### ISSUE TITLE
Write a function that will display wins and losses

### ISSUE THREAD

**Issue by PaulaR-05** [chingu-voyages/v44-tier1-team-05#50]
**__Task Description__**
Leaderboard

- [x]  User can see the display of the total number of wins and losses for each
- [x]  User can see the tally of wins for each of the three ways the game can end
- [x]  User can see the tally of losses for each of the three ways the game can end

**__Technical Considerations__**
Javascript

**__Additional Considerations__**
Any supplemental information including unresolved questions, links to external resources, screenshots, etc.

**Comment by PaulaR-05** [comment]
@goodman2814  
Alex, I'm sorry, I should have add a description here for this task. So, as we discussed, the game will end when one of these conditions are reached:
1. A pre-defined period of time is reached.
2 .A player forfeits two consecutive turns. In this situation, the player loses the game. [I made an issue for it]
3. A player completely fills in their grid. This will take longer than the first two options. Depending on the luck of the roll, players may end up forfeiting many turns.
